src/main/java/stsjorbsmod/cards/cull/Consumption.java

```diff
@@ -0,0 +1,71 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.ui.panels.EnergyPanel;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.actions.ConsumptionAction;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Consumption extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Consumption.class);
+    public static final Logger logger = LogManager.getLogger(Cull.class.getName());
+
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = -1;
+    private static final int DAMAGE = 1;
+
+    public Consumption() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        baseDamage = DAMAGE;
+        this.isMultiDamage = true;
+        this.exhaust = true;
```

Comment:
We'll want to give this the PERSISTENT_POSITIVE_EFFECT card tag to prevent it from showing up worth temporary card generation effects